src/containers/Header/header.scss

```diff
@@ -32,7 +32,6 @@ $toggle-text-color: var(--toggle-text-color);
 
 .preview-flag {
 	font-size: 1rem;
-	color: $toggle-text-color;
+	color: var(--warning);
```

Comment:
@queuing4oranges Dont use `variables` but rather use our predefined constants - meaning here it would be just `$warning` ;)